src/commands/protocols/publish/sender/publish-validate-asset-command.js

```diff
@@ -0,0 +1,29 @@
+import Command from '../../../command.js';
+
+class PublishValidateAssetCommand extends Command {
```

Comment:
Shouldn't this command extend ValidateAssetCommand?

Reply:
Correct, will add